frog/imports/client/StudentView/SessionBody.jsx

```diff
@@ -90,7 +90,7 @@ const StudentView = withRouter(({ activities, session, classes, history }) => (
               className={classes.button}
               color="inherit"
               onClick={() => {}}
-              href="/teacher/orchestration"
+              href="/t/"
```

Comment:
Caution, you are adding a `/` at the end of the URL which wasn't there before. Make sure this is intentional

Reply:
Both are equally valid, will change for consistency